### PR TITLE
Update weather command

### PR DESCRIPTION
**Pull request by Tesutarin**
## Description
Update weather command
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/command/commands/WeatherCommand.java [hunk 1]
```diff
@@ -11,29 +11,30 @@
...
+@Command(label = "weather", usage = "weather <climate> <weather>", aliases = {"w"}, permission = "player.weather", permissionTargeted = "player.weather.others", description = "commands.weather.description")
 public final class WeatherCommand implements CommandHandler {
 
     @Override
     public void execute(Player sender, Player targetPlayer, List<String> args) {
         int weatherId = 0;
         int climateId = 1;
         switch (args.size()) {
-            case 2:
+            case 0:
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/command/commands/WeatherCommand.java]: This won't actually work, you're trying to pull index-0 and index-1 from a size-0 array, and index-1 from a size-1 array.

**Comment by Birdulon** [comment]
Is there some purpose behind swapping the argument order?

**Comment by Tesutarin** [comment]
> Is there some purpose behind swapping the argument order?

nope. just good-looking

**Comment by Tesutarin** [comment]
i'll close this, and open a new pr tomorrow

**Pull request closed without merging**